src/components/NestedScrollView.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {ScrollView, View} from 'react-native';
+import styles from '../styles/styles';
+
+const propTypes = {
+    /** Any children to display */
+    children: PropTypes.node,
+};
+
+const defaultProps = {
+    children: null,
+};
+
+const NestedScrollView = props => (
+    <ScrollView
+        horizontal
+        contentContainerStyle={[styles.flex1, styles.h100, styles.w100]}
+        scrollEnabled={false}
+    >
+        <View style={[styles.w100]}>
+            {props.children}
+        </View>
+    </ScrollView>
+);
```

Comment:
I do not understand the purpose of this component.

Reply:
To make it reusable.  We would have other areas where we have FlatList inside ScrollView, etc. So we can reuse this component to wrap.

Reply:
I think it does not add much value. But its fine.